build-all.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1117
+
+function format_time() {(
+    MINS=$(((${2} - ${1}) / 60))
+    SECS=$(((${2} - ${1}) % 60))
+
+    if [[ ${MINS} -ge 60 ]]; then
+        HOURS=$((MINS / 60))
+        MINS=$((MINS % 60))
+    fi
+
+    [[ -n ${HOURS} ]] && TIME_STRING="${HOURS}h "
+    echo "${TIME_STRING}${MINS}m ${SECS}s"
+)}
+
+BLD_ALL_START=$(date +%s)
+
+echo
+grep "env:" .travis.yml | grep -v LLVM_VERSION | sed "s/.*env: //g" | while read -r ITEM; do (
+    DRIVER_START=$(date +%s)
+    read -ra VALUES <<< "${ITEM}"
```

Comment:
`<<<` will fail on dash shell

I learned this the hard way recently when Android kernel builds were failing, but only due to the build machines using `dash` while the internal "glinux" distros used `bash` (and I use `zsh`).

Reply:
Hmmm, the shebang specifies `bash` so is this a problem? I'll try to look at a more portable solution though.

Reply:
Well, `dash` doesn't do arrays so this entire script will have to be rewritten if being portable is a requirement. `driver.sh` uses some `bash`-isms too, I don't think it is unreasonable to require `bash` for these scripts. Let me know if you feel strongly about that though :)